src/views/Alteration.vue

```diff
@@ -304,6 +307,18 @@ export default class Alteration extends Mixins(
       this.emitFetchError(err)
     }
 
+    // set current profile name to store for field pre population
+    // do this only if we are not staff
+    if (!this.isRoleStaff) {
+      // pre-populate Certified By name
+      this.setCertifyState(
+        {
+          valid: this.getCertifyState.valid,
+          certifiedBy: `${this.getUserFirstName} ${this.getUserLastName}`
+        }
+      )
+    }
+
```

Comment:
Hah. That was easy. Did you test locally as reg user + staff?

Could you add a test case for THIS component to verify that it sets the certify state correctly when staff and not staff?

Reply:
Ya can add to the tests so that it checks that.